hypothesis-python/src/hypothesis/strategies/_internal/numbers.py

```diff
@@ -280,6 +308,7 @@ def floats(
     *,
     allow_nan: Optional[bool] = None,
     allow_infinity: Optional[bool] = None,
+    allow_subnormal: Optional[bool] = None,
```

Comment:
I *like* putting it here in the signature, but it does mean that we'll need to tweak the codemod to ensure that we don't misalign the argument names when refactoring.  I'm happy to push a commit for that before we merge the PR or help you write it as you prefer, it's a fairly esoteric topic 😀

Reply:
Yeah I'll leave work on this PR for next week, so RE codemods feel free to push whatever change you think is ideal.

Reply:
Pushed; we'll want to rebase on #3161 once that's in but this is fine for now.